plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java

```diff
@@ -164,7 +163,7 @@ private T tryConstructor(Constructor<T> constructor, Mapping config, Configurati
                                 new Object[]{target, names[i], t == Secret.class || Attribute.calculateIfSecret(target, names[i]) ? "****" : value});
                     }
                 } else if (t.isPrimitive()) {
-                    args[i] = defaultValue(t);
+                    args[i] = defaultPrimativeValue(t);
```

Comment:
https://en.wiktionary.org/wiki/primative


```suggestion
                    args[i] = defaultPrimitiveValue(t);
```

Reply:
fixed in 95befad5de4bf6c72632f504c62838a7c3757c3a